README.md

```diff
@@ -70,6 +70,26 @@ Used for optimizing images that come from backdrops and emblems.
  * `IMAGE_OPTIMIZE_KEY` - The key for API use.
  * `IMAGE_DOMAIN` - The domain for API Service
 
+### Docker
```

Comment:
We could move this section up to the top and make a duplicate `### Setup (Docker)` section

Reply:
Done.